### DISCUSSION TITLE
Is there a way to output russian text correctly with -j option?

### DISCUSSION THREAD

**Discussion by soredake** [streamlink/streamlink#6079] [Q&A] [answered]
### Description

When doing `streamlink https://www.youtube.com/@Nuke73-Live best -j`:

I got this:
```
"title": "RGG-LAND \u041b\u0415\u0422\u041e 2024 \u2022 \u0414\u0435\u043d\u044c 16 \u2022 \u0427\u0430\u0441\u0442\u044c 1"
```

Is there a way to make streamlink output title correctly? Like `"title": "RGG-LAND ЛЕТО 2024 • День 16 • Часть 1"`

**Comment by bastimeyer** [comment, marked as answer]
Streamlink's CLI defaults to writing ASCII-only characters in its JSON output. Any JSON processor like `jq` for example will usually decode this again.

```
$ streamlink https://www.youtube.com/@Nuke73-Live best -j | jq .metadata
{
  "id": "8q-W_JN7qZg",
  "author": "Nuke73",
  "category": "Gaming",
  "title": "RGG-LAND ЛЕТО 2024 • День 16 • Часть 2"
}
```

We can change it though, as there's no reason to escape unicode characters outside the ASCII range when the output is UTF-8 encoded anyway.
https://github.com/streamlink/streamlink/pull/6080
